packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md

```diff
@@ -24,13 +34,19 @@ The following environment variables can be set to configure the Proxy sample ins
 | -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
 | `PROXY_BUNDLE_PATH`                        | Path to the JSS SPA app's `server.bundle.js`. Default can be seen in [config.js](./config.js) file.                                                                                                                    |
 | `PROXY_PORT`                              | Optional. Port which will be used when start sample. Default can be seen in [config.js](./config.js) file.                                                             |
-| `DEBUG`                  | Optional. Debug level for the proxy. Set the DEBUG environment variable to 'sitecore-jss:*,proxy*,http-proxy-middleware*' to see all logs.                                                                                                                  |
+| `DEBUG`                  | Optional. Debug level for the proxy. Set the DEBUG environment variable to 'sitecore-jss:*,proxy*,http-proxy-middleware*' to see all logs.
```

Comment:
Please specify the possible values (like 'sitecore-jss:layout', 'sitecore-jss:personalize' etc) and what they mean here or in a separate section

Reply:
Added.
i don't think we need to explain their purpose since the general explanation debug and their names should be self explanatory.

Reply:
I'm not sure if it is self explanatory for someone starting out with JSS. You can add a link to our docs here to not duplicate info:
https://doc.sitecore.com/xp/en/developers/hd/latest/sitecore-headless-development/debug-logging-in-jss-apps.html#namespaces